dependencies.yaml

```diff
@@ -112,13 +167,14 @@ dependencies:
           - *gmock
   test_python:
     common:
-      - output_types: [conda, requirements]
+      - output_types: [conda, requirements, pyproject]
         packages:
+          - cloudpickle
           - cudf==23.12.*
           - cupy
           - dask>=2023.1.1
           - distributed>=2023.1.1
```

Comment:
```suggestion
          - dask
          - distributed
```

Reply:
Let's make this [consistent with UCX-Py](https://github.com/rapidsai/ucx-py/blob/2a22a67a4abbafe3facfd5348030b272dfacfa5b/dependencies.yaml#L120-L121) for now. I'll have to review how we pin to Dask, whether we should follow the same practice to pin/unpin as we do for the rest of RAPIDS or otherwise.